### ISSUE TITLE
Can't addthirdpartycamera

### ISSUE THREAD

**Issue by xjz89982** [allenai/ai2thor#327]
Hello, my ai2thor version is 2.1.1. I used following code to add third party camera, as written in documentation page, but got
> TypeError: step() got an unexpected keyword argument 'rotation' 
```
controller.step( action='AddThirdPartyCamera', rotation=dict(x=0, y=90, z=0),position=dict(x=-1.25, y=1.0, z=-1.0),fieldOfView=60)
```
Then i tried to add  third party camera in the old way 
```
event = controller.step(dict(action='AddThirdPartyCamera', rotation=dict(x=0, y=0, z=0), position=dict(x=0, y=1.59705162, z=0), fieldOfView=90))
```
This time it got stuck in controller.step.
Besides, if i use old version executable file `thor-201906111128-Linux64`, it works very well.

**Comment by mattdeitke** [comment]
Hey!

Sorry for the late response. The syntax you are trying to use
```python
controller.step(action='AddThirdPartyCamera', <other kwargs>)
```
was introduced in version 2.2.0.

I'm not sure why the old way gets stuck. If you're still having issues, try updating your version and let me know.

**Comment by engNagi** [comment]
Hey!
regarding the action of adding a third-party camera, on my mac in version 2.0.2 the third party camera is rendered and I can retrieve both the first-person camera frames and the second, but with last version 2.2.0 the action of adding third-party camera does not render anything at all, what I need is to have both frames from the first person and third party cameras but playing of the environment should show third party camera frames. so I can observe the agent, is this already implemented? i f yes how I can correctly use it? if not would it be supported soon?

**Comment by mattdeitke** [comment]
Closing the follow-up discussion as it appears to be solved in #385. Feel free to re-open if there are any related questions :)
